modules/cluster-logging-upgrading-elasticsearch.adoc

```diff
@@ -168,28 +144,29 @@ green  open   audit-000001
 ----
 ====
 
-.. Verify that the log collector is updated:
+. Verify that the log collector is updated by entering the following commands and observing the output:
```

Comment:
This is unrelated to upgrading elasticsearch and was more appropriate when we talked solely about the entire logging stack in a single upgrade document.  The only relevent bit may be that the EO controls the certificates to use to talk to ES.  This only significant when/if EO rotates them and the collector should restart to make use of the new certs.  Not really certain how that fits into this flow

Reply:
I can remove this if it makes sense to do so?

Reply:
I would lean towards removing it all together... especially since we are purely talking about Elasticsearch here